q2_composition/plugin_setup.py

```diff
@@ -104,3 +103,11 @@
             ex.ancombc_multi_formula_with_reference_levels)
     }
 )
+
+plugin.visualizers.register_function(
+    function=q2_composition.tabulate,
+    inputs={'dataloaf': FeatureData[DifferentialAbundance]},
```

Comment:
Maybe call this parameter `data` instead? Dataloaf won't mean much to end-users.